### PR TITLE
[LLM_BENCH] Whisper pipeline with Optimum-Intel API

### PR DESCRIPTION
**Pull request by wgzintel**
whisper speech gen:
[whisper.csv](https://github.com/user-attachments/files/17366402/whisper.csv)
[whisper.json](https://github.com/user-attachments/files/17366404/whisper.json)
[whisper_log.txt](https://github.com/user-attachments/files/17370614/whisper_log.txt)

qwen2-0.5b text gen:
[qwen2-0.5b.json](https://github.com/user-attachments/files/17366411/qwen2-0.5b.json)
[qwen2-0.5b.csv](https://github.com/user-attachments/files/17366412/qwen2-0.5b.csv)
[qwem2-0.5b_log.txt](https://github.com/user-attachments/files/17370611/qwem2-0.5b_log.txt)

stable-diffusion image gen:
[stable-diffusion.json](https://github.com/user-attachments/files/17370616/stable-diffusion.json)
[stable-diffusion.csv](https://github.com/user-attachments/files/17370617/stable-diffusion.csv)
[stable-diffusion_log.txt](https://github.com/user-attachments/files/17370618/stable-diffusion_log.txt)

### CODE REVIEW

File: llm_bench/python/llm_bench_utils/ov_utils.py [hunk 1]
```diff
@@ -271,6 +271,34 @@ def create_ldm_super_resolution_model(model_path, device, **kwargs):
...
+    model_type = kwargs.get('model_type', default_model_type)
+    model_class = OV_MODEL_CLASSES_MAPPING.get(model_type, OV_MODEL_CLASSES_MAPPING[default_model_type])
+    model_path = Path(model_path)
+    model_path_existed = model_path.exists()
+    # load model
+    if not model_path_existed:
+        raise RuntimeError(f'==Failure ==: model path:{model_path} does not exist')
+    else:
+        start = time.perf_counter()
+        ov_model = model_class.from_pretrained(
```
Comment (andrei-kochin) [inline llm_bench/python/llm_bench_utils/ov_utils.py:291]: Why do we have optimum here? it is supposed to be the the whisper pipeline form GenAI

**Review by andrei-kochin — CHANGES_REQUESTED** [review]

File: llm_bench/python/llm_bench_utils/ov_utils.py (hunk 1 above)
Comment (peterchen-intel) [reply llm_bench/python/llm_bench_utils/ov_utils.py:299]: @andrei-kochin both will be supported.

Comment (peterchen-intel) [reply llm_bench/python/llm_bench_utils/ov_utils.py:299]: Yes.

**Comment by ilya-lavrenov** [comment]
is it planned for 2024.5 ?

**Comment by peterchen-intel** [comment]
Let's focus on OV genai API for now.  https://github.com/openvinotoolkit/openvino.genai/pull/1025

**Comment by peterchen-intel** [comment]
@wgzintel Please reference to https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/distil-whisper-asr/distil-whisper-asr.ipynb

**Comment by peterchen-intel** [comment]
merged to https://github.com/openvinotoolkit/openvino.genai/pull/1025

**Pull request closed without merging**